packages/styleguide/.env.local

```diff
@@ -0,0 +1 @@
+CONTAINER_API_BASE=propeller.cc-le-cf.com
```

Comment:
why are we using the prod endpoint in local? and can you set up the staging and prod ones too?

Reply:
@borisonr jake mentioned that since this is open source, then we want users to only be aware of our prod endpoint, not our staging one. 

if we really do want to set up a staging endpoint here, which we would only need once we build a gamut-like storybook experience on a server, we'd probably need to do this in circleci or github actions. else, it's not essential. 

for the current purposes, i think this covers what we need! cc'ing @jakemhiller to get some more input, too! 

(eventually, the parent/container calling codebytes will pass in a staging/prod endpoint of its own)

Reply:
thats fine! but maybe we dont call this file local then? just have one env file

Reply:
totally reasonable!